content/pt-br/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,291 @@
+
+* Você pode verificar o endereço MAC da interface de rede utilizando o comando `ip link` ou o comando `ipconfig -a`.
+* O product_uuid pode ser verificado utilizando o comando `sudo cat /sys/class/dmi/id/product_uuid`.
+
+É provável que dispositivos físicos possuam endereços únicos. No entanto, é possível que algumas máquinas virtuais possuam endereços iguais. O Kubernetes utiliza esses valores para identificar unicamente os nós em um cluster. Se esses valores não forem únicos para cada nó, o processo de instalação pode [falhar](https://github.com/kubernetes/kubeadm/issues/31).
+
+## Verificando os adaptadores de rede
+
+Se você possuir mais de um adaptador de rede, e seus componentes Kubernetes não forem acessíveis através da rota padrão, recomendamos adicionar o IP das rotas para que os endereços do cluster Kubernetes passem pelo adaptador correto.
+
+## Fazendo com que o iptables enxergue o tráfego agregado
+
+Se assegure de que o módulo `br_netfilter` está carregado. Isso pode ser feito executando o comando `lsmod | grep br_netfilter`. Para carrega-lo explicitamente execute `sudo modprobe br_netfilter`.
+
+Você também deve se assegurar de que a configuração `net.bridge.bridge-nf-call-iptables` no seu `sysctl` está com o valor 1. Um requerimento para que o iptables de cada um dos seus nós Linux enxergue o tráfego agregado corretamente. Como no exemplo abaixo:
+
+```bash
+cat <<EOF | sudo tee /etc/modules-load.d/k8s.conf
+br_netfilter
+EOF
+
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-ip6tables = 1
+net.bridge.bridge-nf-call-iptables = 1
+EOF
+sudo sysctl --system
+```
+
+Para mais detalhes veja a página [Requisitos do plugin de rede](/pt-br/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements).
+
+## Verificando as portas necessárias
+
+### Nós da camada de gerenciamento
+
+| Protocolo | Direção | Intervalo de portas | Propósito       | Utilizada por             |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Entrada   | 6443\*     | API do Kubernetes       | Todos                      |
+| TCP      | Entrada   | 2379-2380  | API servidor cliente do etcd | kube-apiserver, etcd      |
+| TCP      | Entrada   | 10250      | API do kubelet          | kubeadm, Camada de gerenciamento       |
+| TCP      | Entrada   | 10251      | kube-scheduler          | kubeadm                      |
+| TCP      | Entrada   | 10252      | kube-controller-manager | kubeadm                      |
+
+### Nós de processamento
+
+| Protocolo | Direção | Intervalo de portas | Propósito       | Utilizada por             |
+|----------|-----------|-------------|-----------------------|-------------------------|
+| TCP      | Entrada   | 10250       | API do kubelet        | kubeadm, Camada de gerenciamento     |
+| TCP      | Entrada   | 30000-32767 | Serviços NodePort†    | Todos                     |
+
```

Comment:
This in English points to: https://kubernetes.io/docs/reference/ports-and-protocols/ 
```
Check required ports

These [required ports](https://kubernetes.io/docs/reference/ports-and-protocols/ ) need to be open in order for Kubernetes components to communicate with each other.

The pod network plugin you use (see below) may also require certain ports to be open. Since this differs with each pod network plugin, please see the documentation for the plugins about what port(s) those need
```

Please add the same approach into this translation

Reply:
Yeah, actually this change was made four days ago. When I did the translation there was this table instead, and the "ports and protocols" page did not exist. But I'm glad they changed it, it seemed like too much information on that same page. Changes done to reflect this new state.